### PR TITLE
Add Start Menu

### LINKED ISSUE #32: Add Start Menu
Start menu to:

* select playing mode (offline/online)
* select side (light/dark)
* introduce how to play the game (key presses/mouse uses)

### PR DESCRIPTION
**Pull request by Muazat**
I added an initial menu with options; Online, offline, side and Quit. Th e offline option has four options ;Single player, Two players, Ai mode and Quit. The single player is not linked to anything yet. The two players is linked to the board. It has all other functionalities of the previous state except you cannot switch to AI mode while playing. The AI mode option displays an AI vs AI game while the quit option terminates the game.
All the above mentioned options can be selected by double clicking on them.

### CODE REVIEW

**Review by Mrsatatima — CHANGES_REQUESTED** [review]
good job.... but main_menu.py and menu.py have same contents... what is the difference.. if no difference delete one...
also after making the said changes pull from the upstream because it has been updated... this is to avoid merge conflict when merging...
make all changes and push.. welcome to steam-chess contributors

**Comment by Muazat** [comment]
The main_menu and menu are two different menus. After reading the description of the add start menu, I came to realize that the Menu did not meet the description. So I had to add the main_menu that will link to the menu.

**Comment by drak00** [comment]
please fix merge conflicts so files can be merged

File: menu.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+font = pg.font.SysFont("Helvetica",75)
+#
+def display_main(choice):
+	"""
+		Link option To main board
+	"""
+	return main(choice)
+
+
+def player_format(message, playerSize, playerColor):
```
Comment (drak00) [inline menu.py]: coming from java, I can understand your proclivity to using camel case, however to conform to the project's coding style which uses a more modern python convention, pls use underscores "_" to separate variable names: so playerSize would be player_size

File: menu.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+				elif (460 <= a <= 550) and (500<=b<=530):
+					if selected !="quit":
+						selected="quit"
+					elif selected =="quit":
+						mode="quit"
+						
+		# Main Menu UI
+		# screen.fill(yellow)
+		title=player_format("STEAM CHESS ENGINE", 100, yellow)
+		name=pg.transform.scale(pg.image.load("images/menu.jfif"),(640,640)) #background image
```
Comment (drak00) [inline menu.py]: do ensure this image is not copyrighted!

**Review by drak00 — CHANGES_REQUESTED** [review]
Nice work! Please make the corrections so we can merge!

File: menu.py (hunk 1 above)
Comment (Muazat) [reply menu.py]: Noted

**Pull request merged**